tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:path/path.dart' as path;
+import 'package:process_runner/process_runner.dart';
+
+import '../build_utils.dart';
+
+import 'command.dart';
+
+const String _configFlag = 'config';
+
+// Some wrinkles I'd like to iron out:
+//  - There can be multiple GlobalBuild objects with the same name. How
+//    should we deal with these conflicts? "mac_clang_tidy" and
+//    "mac_host_engine" both define "host_debug".
+//  - Naming: Should BuildConfig be BuilderConfig and GlobalBuild be BuildConfig?
+//  - The BuildConfig database seems to be lacking things I expected:
+//    - There is no "android_debug_*" GlobalBuild in any of the builders that run on mac?
+//    - The gn flags specified in the GlobalBuilds don't reference rbe or goma? Should this be a local configuration that the user selects?
```

Comment:
The `gn` script and CI try to enable Goma implicitly, so the flag doesn't show up too much in the CI configs.

The migration to RBE is in-progress, but there are a few builds that have it. It has to be requested explicitly both locally and in CI.